.github/workflows/terraform-apply.yml

```diff
@@ -116,6 +116,63 @@ jobs:
         run: terraform apply -input=false -no-color terraform.tfplan
         continue-on-error: true
 
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+          cache: "pip" # caching pip dependencies
+      - run: pip install -r $GITHUB_WORKSPACE/scripts/requirements.txt
+
+      - name: Export Terraform State to JSON
+        id: read_state
+        run: terraform show -json > state.json
+
+      - name: Extract Breakglass Roles
```

Comment:
This bash is kind of gross, I want to see if we can put it in Python for better readability, I'll take a stab at it.

Reply:
bash is not gross, but I added your changes to the python script in the new commit.